### PR TITLE
Remove testnet from docker command examples

### PR DESCRIPTION
**Pull request by ArturWieczorek**

### CODE REVIEW

**Review by kderme — DISMISSED** [review]
LGTM

File: doc/docker.md [hunk 1]
```diff
@@ -25,11 +25,6 @@ The PostgreSQL database is exposed on localhost port `5432`
 
 `$ psql -h 0.0.0.0 -p 5432 -U postgres -d cexplorer` (then enter secret password)
 
-### To connect to another network:
-```
-NETWORK=testnet docker-compose up && docker-compose logs -f
```
Comment (jbgi) [inline doc/docker.md:30]: instead, once #1253 is merge, we can replace `testnet` with `preprod` (or `preview`).

Comment (ArturWieczorek) [reply doc/docker.md:30]: Sure :+1:  @jbgi

**Pull request merged**